src/franka_plan_runner.cc

```diff
@@ -267,34 +255,38 @@ int FrankaPlanRunner::RunFranka() {
 
         // prevent the plan from being started if robot is not running...
         if (comm_interface_->HasNewPlan() && status_ == RobotStatus::Running) {
-          dexai::log()->info("RunFranka: Got a new plan, attaching callback!");
+          dexai::log()->info(
+              "RunFranka: found a new plan, attaching callback...");
           status_has_changed = true;
-          // joint_position_callback_ or impedance_control_callback_ can be used
-          // here:
-          robot.control(joint_position_callback_);
+          // Use either joint position or impedance control callback here
+          robot.control(std::bind(&FrankaPlanRunner::JointPositionCallback,
+                                  this, std::placeholders::_1,
+                                  std::placeholders::_2));
```

Comment:
What are these placeholders?

Reply:
the two arguments of the callback function. this gets rid of the lambda variable.